src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/context/HttpPostContextTest.kt

```diff
@@ -0,0 +1,46 @@
+package io.github.rybalkinsd.kohttp.dsl.context
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+/**
+ * @author doyaaaaaken
+ */
+class HttpPostContextTest {
```

Comment:
As I said in comment to pull request, it would be great if you will also add integration tests.